commands/rm.go

```diff
@@ -70,23 +71,28 @@ func rmCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVar(&options.keepState, "keep-state", false, "Keep BuildKit state")
+	flags.BoolVar(&options.keepBuildkitd, "keep-buildkitd", false, "Keep the buildkitd daemon running")
```

Comment:
I think `--keep-instance` would be a better name.

Reply:
@crazy-max maybe `--keep-daemon`? "Instance" is not very descriptive either as anything can have "an instance".

Reply:
yes `--keep-daemon` sgtm :+1:

Reply:
`--keep-daemon` sgtm too 👍 

I'll make the change